src/main/java/org/radarbase/appserver/service/protocol/GithubClient.java

```diff
@@ -73,6 +74,11 @@ public String getGithubContent(String url) throws IOException, InterruptedExcept
         if (isSuccessfulResponse(response)) {
             return response.body().toString();
         } else {
+            if (response.statusCode() == 301) {
```

Comment:
You can probably just use `setRedirectStrategy(new LaxRedirectStrategy())` on the HTTP client builder if the httpclient version is recent enough.

Reply:
Thanks I didn't know about this.